### PR TITLE
Add PHP Coding Standards Fixer in CI

### PR DESCRIPTION
**Pull request by jbelien**
In the same logic as adding PHPStan (see #1193) to make Geocoder PHP code more clean and consistent across the different providers, plugins, ... I suggest to also implement [PHP Coding Standards Fixer](https://cs.symfony.com/) with [`@Symfony` ruleset](https://cs.symfony.com/doc/ruleSets/Symfony.html).

### CODE REVIEW

**Comment by norkunas** [comment]
I'd suggest to use [php-cs-fixer/shim](https://packagist.org/packages/php-cs-fixer/shim) :slightly_smiling_face:

File: composer.json [hunk 1]
```diff
@@ -82,6 +83,8 @@
     },
     "scripts": {
         "analyse": "vendor/bin/phpstan analyse",
+        "cs": "vendor/bin/php-cs-fixer fix -v --dry-run --rules=@Symfony src/",
+        "cs:fix": "vendor/bin/php-cs-fixer fix -v --rules=@Symfony src/",
```
Comment (norkunas) [inline composer.json:87]: I think it's better to create a `.php-cs-fixer.dist.php` with `@Symfony` ruleset definition, so running locally no need to provide rules

**Comment by jbelien** [comment]
> I'd suggest to use [php-cs-fixer/shim](https://packagist.org/packages/php-cs-fixer/shim) 🙂

Oh, I didn't know that. I'll make the change! 👍

File: composer.json (hunk 1 above)
Comment (jbelien) [reply composer.json]: Done in 785e21c6219b9388cb4a444abf33263c78125806

**Review by norkunas — APPROVED** [review]

**Pull request merged**